<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewInstance</class>
 <widget class="QDialog" name="DialogNewInstance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Instance</string>
  </property>
  <layout class="QVBoxLayout" name="vertical_layout">
   <item>
    <layout class="QFormLayout" name="layout_name_input">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Group</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edit_name">
       <property name="minimumSize">
        <size>
         <width>137</width>
         <height>22</height>
        </size>
       </property>
       <property name="maxLength">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="combo_box_group">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
